cpp/src/io/parquet/reader_impl_chunking.cu

```diff
@@ -114,13 +130,13 @@ void print_cumulative_row_info(host_span<cumulative_row_info const> sizes,
 #endif  // CHUNKING_DEBUG
 
 /**
- * @brief Functor which reduces two cumulative_row_info structs of the same key.
+ * @brief Functor which reduces two cumulative_page_info structs of the same key.
  */
-struct cumulative_row_sum {
-  cumulative_row_info operator()
-    __device__(cumulative_row_info const& a, cumulative_row_info const& b) const
+struct cumulative_page_sum {
```

Comment:
Should it be `page_prefix_sum`? Because `cummulative_` is essentially `prefix_`.

Reply:
It could be, but I'm using `cumulative` elsewhere in here. In particular:  `cumulative_page_info` to represent cumulative row counts and byte sizes for groups of pages.  'prefix sum' doesn't seem as clear to me.